src/components/toolbar/type-selector/TypeSelector.tsx

```diff
@@ -74,40 +75,40 @@ const buildOptions = (state: EditorState): Option[] => {
       const beforeSection = $from.before(sectionDepth)
       const $beforeSection = doc.resolve(beforeSection)
       const sectionOffset = $beforeSection.parentOffset
-
-      options.push(
-        {
+      const section = $from.node(sectionDepth)
+      !isDeleted(section) &&
```

Comment:
Is the intention to remove the option or disable it? I see you are removing it here!